src/api/db/structure.sql

```diff
@@ -11,7 +11,7 @@ CREATE TABLE `architectures` (
   `name` varchar(255) CHARACTER SET utf8 NOT NULL,
   `available` tinyint(1) DEFAULT '0',
   PRIMARY KEY (`id`),
-  UNIQUE KEY `arch_name_index` (`name`) USING BTREE
+  UNIQUE KEY `arch_name_index` (`name`)
```

Comment:
There is a reason why this commit was named "update structure.sql for now";-)
I'll change it later.